src/models/vehicle-info.ts

```diff
@@ -69,7 +69,11 @@ export class VehicleInfo extends BaseModel {
     }
 
     public getConstructionDateTimestamp() {
-        return new Date(this.getConstructionYear(), this.getConstructionMonth(), this.getConstructionDay()).getTime();
+        return new Date(
+            this.getConstructionYear(),
+            this.getConstructionMonth() - 1,
```

Comment:
kan je dit niet beter in de functie `getConstructionMonth()` zelf aanpassen? Nu is het alsnog fout als je de maand ergens anders wilt aanroepen